### ISSUE TITLE
v2 has no anchors?

### ISSUE THREAD

**Issue by zouliangyu** [Stinky-Tofu/YOLO_V3#39]

**Comment by Stinky-Tofu** [comment]
@zouliangyu Yes, It's my idea. And it's very similar to the idea of this [paper(CVPR 2019)](https://arxiv.org/abs/1903.00621).

**Comment by Stinky-Tofu** [comment]
@zouliangyu Yes, It's my idea. And it's very similar to the idea of this [paper(CVPR 2019)](https://arxiv.org/abs/1903.00621).
